symengine/functions.cpp

```diff
@@ -2469,6 +2469,74 @@ RCP<const Basic> erfc(const RCP<const Basic> &arg)
     return make_rcp<const Erfc>(d);
 }
 
+bool Erfinv::is_canonical(const RCP<const Basic> &arg) const
+{
+    if (eq(*arg, *zero) or eq(*arg, *one))
+        return false;
+    if (could_extract_minus(*arg))
+        return false;
+    return true;
+}
+
+RCP<const Basic> Erfinv::create(const RCP<const Basic> &arg) const
+{
+    return erfinv(arg);
+}
+
+RCP<const Basic> erfinv(const RCP<const Basic> &arg)
+{
+    if (eq(*arg, *zero)) {
+        return zero;
+    }
+    if (eq(*arg, *one)) {
+        return Inf;
+    }
+    if (eq(*arg, *minus_one)) {
+        return NegInf;
+    }
+
+    RCP<const Basic> d;
+    bool b = handle_minus(arg, outArg(d));
+    if (b) {
+        return neg(erfinv(d));
+    }
+    return make_rcp<const Erfinv>(d);
+}
+
+bool Erfcinv::is_canonical(const RCP<const Basic> &arg) const
+{
+    if (eq(*arg, *one) or eq(*arg, *integer(2)) or eq(*arg, *zero)) {
+        return false;
+    }
+    if (could_extract_minus(*arg))
+        return false;
+    return true;
+}
+
+RCP<const Basic> Erfcinv::create(const RCP<const Basic> &arg) const
+{
+    return erfcinv(arg);
+}
+
+RCP<const Basic> erfcinv(const RCP<const Basic> &arg)
+{
+    if (eq(*arg, *zero)) {
```

Comment:
You can also check that the input is a `Erfc` instance and return the arg of `Erfc`